.rubocop.yml

```diff
@@ -1,80 +1,199 @@
+  - lib/jekyll/renderer.rb
+  - lib/jekyll/site.rb
+  - lib/jekyll/static_file.rb
+  - lib/jekyll/stevenson.rb
+  - lib/jekyll/tags/highlight.rb
+  - lib/jekyll/tags/include.rb
+  - lib/jekyll/tags/link.rb
+  - lib/jekyll/tags/post_url.rb
+  - lib/jekyll/theme.rb
+  - lib/jekyll/url.rb
+  - lib/jekyll/utils/ansi.rb
+  - lib/jekyll/utils/platforms.rb
+  - lib/jekyll/utils.rb
+  - lib/jekyll/version.rb
+  - lib/jekyll.rb
+  - bin/**/*
   - script/**/*
-  - spec/**/*
+  - vendor/**/*
+Lint/AmbiguousRegexpLiteral:
+  Exclude:
+  - features/step_definitions.rb
+Lint/EndAlignment:
+  Severity: error
+Lint/UnreachableCode:
+  Severity: error
+Lint/UselessAccessModifier:
+  Enabled: false
+Metrics/AbcSize:
+  Max: 20
+Metrics/ClassLength:
+  Max: 240
+  Exclude:
+  - !ruby/regexp /features\/.*.rb$/
+Metrics/CyclomaticComplexity:
+  Max: 8
+Metrics/LineLength:
+  Max: 90
+  Severity: warning
+  Exclude:
+  - !ruby/regexp /features\/.*.rb/
+Metrics/MethodLength:
+  Max: 20
+  CountComments: false
+  Severity: error
+Metrics/ModuleLength:
+  Max: 240
+Metrics/ParameterLists:
+  Max: 4
+Metrics/PerceivedComplexity:
+  Max: 8
+Style/Alias:
+  Enabled: false
+Style/AlignArray:
+  Enabled: false
+Style/AlignHash:
+  SupportedLastArgumentHashStyles: always_ignore
+Style/AlignParameters:
+  EnforcedStyle: with_fixed_indentation
+  Enabled: false
+Style/AndOr:
+  Severity: error
+Style/Attr:
+  Enabled: false
+Style/BracesAroundHashParameters:
+  Enabled: false
+Style/ClassAndModuleChildren:
+  Enabled: false
+Style/Documentation:
+  Enabled: false
+  Exclude:
+  - !ruby/regexp /features\/.*.rb$/
+Style/DoubleNegation:
+  Enabled: false
+Style/EmptyLinesAroundAccessModifier:
+  Enabled: false
+Style/EmptyLinesAroundModuleBody:
+  Enabled: false
+Style/ExtraSpacing:
+  AllowForAlignment: true
+Style/FileName:
+  Enabled: false
+Style/FirstParameterIndentation:
+  EnforcedStyle: consistent
+Style/GuardClause:
+  Enabled: false
+Style/HashSyntax:
+  EnforcedStyle: hash_rockets
+  Severity: error
+Style/IfUnlessModifier:
+  Enabled: false
+Style/IndentArray:
+  EnforcedStyle: consistent
+Style/IndentHash:
+  EnforcedStyle: consistent
+Style/IndentationWidth:
+  Severity: error
+Style/ModuleFunction:
+  Enabled: false
+Style/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+Style/MultilineOperationIndentation:
+  EnforcedStyle: indented
+Style/MultilineTernaryOperator:
+  Severity: error
+Style/PercentLiteralDelimiters:
+  PreferredDelimiters:
+    "%q": "{}"
+    "%Q": "{}"
+    "%r": "{}"
+    "%s": "()"
+    "%w": "()"
+    "%W": "()"
+    "%x": "()"
+Style/RedundantReturn:
+  Enabled: false
+Style/RedundantSelf:
+  Enabled: false
+Style/RegexpLiteral:
+  EnforcedStyle: slashes
+Style/RescueModifier:
+  Enabled: false
+Style/SignalException:
+  EnforcedStyle: only_raise
+Style/SingleLineMethods:
+  Enabled: false
+Style/SpaceAroundOperators:
+  Enabled: false
+Style/SpaceInsideBrackets:
+  Enabled: false
+Style/StringLiterals:
+  Enabled: false
```

Comment:
I would love if we kicked this on, I like Github have a huge love of "" over '' because I find that I interpolate more than I don't but I'll digress that I don't even know if Github themselves enforces it in their own source.